chapters/7-relationships-between-SPDX-elements.md

```diff
@@ -1,6 +1,6 @@
-# 7 Relationships between SPDX Elements
+# Relationships between SPDX elements fields
```

Comment:
"elements fields" doesn't make  sense.
Lets go with "Relationships between SPDX fields".    Elements is in the model, but not really visible in the spec at this time.

Reply:
applying this for now, and we'll resolve as part of issue #347